src/Storefront/Resources/views/storefront/component/product/card/price-unit.html.twig

```diff
@@ -81,10 +85,29 @@
                 </span>
                 {% if isRegulationPrice %}
                     <span class="product-price with-regulation-price">
-                        {% if isListPrice %}<br>{% endif %}<span class="regulation-price">{{ 'general.listPricePreviously'|trans({'%price%': price.regulationPrice.price|currency }) }}{{ 'general.star'|trans|sw_sanitize }}</span>
+                        {# @deprecated tag:v6.7.0 - Showing asterisk next to every price is deprecated. Tax and shipping info is displayed as text instead if `allowBuyInListing` is true. #}
+                        {% if isListPrice %}<br>{% endif %}<span class="regulation-price">{{ 'general.listPricePreviously'|trans({'%price%': price.regulationPrice.price|currency }) }}{% if not feature('ACCESSIBILITY_TWEAKS') %}{{ 'general.star'|trans|sw_sanitize }}{% endif %}</span>
                     </span>
                 {% endif %}
             </div>
         {% endblock %}
+
+        {#
+            If a product can be added to the shipping directly from the listing,
+            we need to display information about taxes and shipping.
+        #}
+        {% if feature('ACCESSIBILITY_TWEAKS') and config('core.listing.allowBuyInListing') %}
+            {% block component_product_box_price_tax_info %}
```

Comment:
Block could be added to the changelog. It could be relevant for developers to overwrite this if they don't use our standard shop pages.

Reply:
Done 👍